src/pages/javascript/index.md

```diff
@@ -28,4 +28,6 @@ Chakra, a fork of Jscript, is developed by Microsoft and used in their Edge brow
 
 Mozilla also offers Rhino, an implementation of JavaScript built in Java, typically embedded into Java applications to provide scripting to end users.
 
-WebKit (except for the Chromium project) implements the JavaScriptCore engine.
\ No newline at end of file
+WebKit (except for the Chromium project) implements the JavaScriptCore engine.
+ 
+Other most frequently used JavaScript Frameworks are React JS, Angular JS, Create JS, jQuery etc. 
```

Comment:
Hi @shashwat1998 

You are including JavaScript frameworks in the JavaScript Engines. Can you please remove it as JS Engines and JS Frameworks are not the same thing?